packages/cli/src/util/output/create-output.ts

```diff
@@ -164,4 +184,28 @@ export class Output {
 
     return promise;
   };
+
+  /**
+   * Returns an ANSI formatted hyperlink when support has been enabled.
+   */
+  link = (
+    text: string,
+    url: string,
+    { fallback }: LinkOptions = {}
+  ): string => {
+    // Credit: https://github.com/sindresorhus/terminal-link
```

Comment:
I think we should start following a convention that's something like [how @styfle and I attributed part of the `vc env pull` code](https://github.com/vercel/vercel/pull/8170/files#diff-32c1018d233f2c92e3dd39a35fa9abe29eeb1e9dc26d49e603b22023e860fb83). Something like

```
Originally authored by Sindre Sorhus under [license]: [link to license]
[link to piece of code]
```

Reply:
This doesn't seem like a credit-worth amount or significance of code to me, but I'm happy to include it if anyone would like to. If we keep it, an expanded comment (as Matthew mentioned) would probably be best.